themes/classic/templates/catalog/_partials/product-prices.tpl

```diff
@@ -79,7 +79,7 @@
 
     {block name='product_ecotax'}
       {if $product.ecotax.amount > 0}
-        <p class="price-ecotax">{l s='Including %amount% for ecotax' d='Shop.Theme.Catalog' sprintf=['%amount%' => $product.ecotax_tax_inc]}
+        <p class="price-ecotax">{l s='Including %amount% for ecotax' d='Shop.Theme.Catalog' sprintf=['%amount%' => $product.ecotax_tax_inc]} {$currency.sign}
```

Comment:
I think we should add the currency sign following how other variables carry it (like `$product.price_tax_exc`). I think these other variables use something like `formatPrice()` ? 🤔